site/docs/patterns/split-button.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Split button
+layout: DetailPattern
+aliases:
+  - /salt/patterns/split-button
+data:
+  components: ["Button", "Menu", "Segmented Button Group", "Tooltip"]
+---
+
+A Split Button enables users to perform a main action and have access to relevant, supplementary actions displayed inside the menu. 
+
+<Diagram
+  src="/img/patterns/split-button/split-button-hero-shot.png"
+  alt="Example showing a split button."
+  border
+/>
+
+## When to use
+
+Use the pattern when providing users with an action that has multiple commands associated with it. Users are able to immediately action the default command and access the supplementary actions, which include more specific or less commonly-used actions.
+
+For example, “Paste” functionality commands could be split as follows: 
+- Default command: Paste 
+- Alternative commands: Keep source formatting, Match formatting, Paste special...
+
+## How to build
+
+### Anatomy
+
+A split button contains:
+
+1. *Main action:* the default action a user can take.
+2. *Arrow button:* contains a chevron-down icon and triggers the menu.
+3. *Supplementary actions:* lower priority actions displayed in menu. 
+
+<Diagram
+  src="/img/patterns/split-button/split-button-anatomy.png"
+  alt="Split button showing supplementary actions."
+  border
+/>
+
+### Layout
+
+- Use the segmented button group component as the base structure of the split button.
+- Use the chevron-down icon for the menu trigger.
+- Use a menu component to display supplementary actions for users to take.
+- By default, the menu is displayed bottom-end of the arrow button.
+
+<Diagram
+  src="/img/patterns/split-button/split-button-layout.png"
+  alt="layout"
+  border
+/>
+
+<ExampleContainer type="positive" customPillText="Best practices">
+- By default, the menu is displayed bottom-end of the arrow button.
+</ExampleContainer>
```

Comment:
```suggestion
```
as it's not advice for the user, i think we can leave it out of best practices

Reply:
I think "default" is weird, it's something the developer will have to set in code but still not sure if it's a best practise I guess

Reply:
ah, in that case then i can see why we would call it a best practice

Reply:
Are you happy keeping it but rewording it?

Reply:
```suggestion
<ExampleContainer type="positive" customPillText="Best practices">
Display the menu below the arrow button.
</ExampleContainer>
```
how about this? if there's any more detail we can add about what the reader needs to do we can include. also we can delete the line above